tools/MDAnalysis/rdf.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="mda_RDF_analysis" name="RDF Analysis" version="@VERSION@">
```

Comment:
please use only small letters for idea if possible.
Also the other tools we removed the _analysis suffix.

Reply:
Thanks, @bgruening  I will make these changes now.

Reply:
Done!